components/tools/Filters.js

```diff
@@ -0,0 +1,107 @@
+import { useState, useContext } from 'react';
+import {twMerge} from 'tailwind-merge'
+import {ToolFilterContext} from '../../context/ToolFilterContext'
+import ArrowDown from '../icons/ArrowDown';
+import FiltersDropdown from './FiltersDropdown';
+import Button from '../buttons/Button'
+import {languagesColor, technologiesColor} from '../../scripts/tools/tags-color'
+
+export default function Filters({setOpenFilter}) {
+  const {isPaid, isAsyncAPIOwner, languages, technologies, setLanguages, setTechnologies, setisPaid, setAsyncAPIOwner} = useContext(ToolFilterContext)
+  const [openLanguage, setopenLanguage] = useState(false)
+  const [openTechnology, setopenTechnology] = useState(false)
+  const [checkPaid, setCheckPaid] = useState(isPaid)
+  const [checkedLanguage, setCheckedLanguage] = useState(languages)
+  const [checkedTechnology, setCheckedTechnology] = useState(technologies)
+  const [checkOwner, setCheckOwner] = useState(isAsyncAPIOwner)
+
+  const handleApplyFilters = () => {
+    setLanguages(checkedLanguage);
+    setTechnologies(checkedTechnology)
+    setisPaid(checkPaid)
+    setAsyncAPIOwner(checkOwner)
+    setOpenFilter({
+      filter: false,
+      category: false
+    })
+  }
+
+  const clearFilters =() => {
+    setLanguages([]);
+    setTechnologies([])
+    setisPaid(false)
+    setAsyncAPIOwner(false)
+    setOpenFilter({
+      filter: false,
+      category: false
+    })
+  }
```

Comment:
I guess that `Clear filters` is only visible in first dropdown, so if I check some categories (in the second dropdown) and I wanna clear them I need to open first, yes? Very strange, we should add `Clear filters` button in the second dropdown :)

Reply:
So, according to you, the `Category` filter should come inside `Filters` menu? If we make it this way, then what should be replaced by that menu in the bar?